adapters/controllers/basic_controller.go

```diff
@@ -0,0 +1,61 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+	"wizeline/ghibli/services/repository"
```

Comment:
For legibility, I would recommend you to separate the std imports from your own packages like this:

```suggestion
	"encoding/json"
	"net/http"

	"wizeline/ghibli/services/repository"
```

As well as adding the full repository to your package import.

Reply:
Thanks, this was corrected and rebased from the first deliverable.